docs/modules/ROOT/pages/references/quality-requirements/reliability/portal-api-data-recoverability.adoc

```diff
@@ -0,0 +1,21 @@
+= Portal API Ephemeral Data
+
+Source::
+Portal API
+
+Stimulus::
+Data is stored
+
+Environment::
+Portal API
+
+Artifact::
+Portal API and external third-party systems
+
+Response::
+Portal API stores the data in an ephemeral storage while data persistency is granted by the third-party backend system
+
+Rationale::
+The Portal API doesn't store any relevant data persistent.
+All relevant data is available and stored in third-party backend systems.
+The source of truth of the data is outside of the Portal API.
```

Comment:
```suggestion
= Data compatibility with external components

Source::
{product} administrator

Stimulus::
Reinstall APPUiO Global from scratch

Environment::
{zone}

Artifact::
Portal API, external third-party component

Response::
Entities that have their lifecycle outside of {product} have their availability restored

Response measure::
Following entities are recovered at latest after 48 hours since reinstallation:
+
* Invoices

Rationale::
{product} doesn't store any relevant third-party entities.
Although some form of caching is permissible, the cache can be rebuilt entirely in the given timeframe by reintegrating the third-party component.
The source of truth of the mentioned entities is outside of {product}.
```